client-js/app/elements/labrad-plot.ts

```diff
@@ -1,9 +1,17 @@
 import 'd3';
-
+import * as three from 'three';
 import {viridisData} from '../scripts/colormaps';
 import * as datavault from "../scripts/datavault";
 
+/**
+ * Import the THREE namespace from the module.
+ */
+const THREE = three.default;
```

Comment:
I think I will leave this line as-is, then submit another PR making the required typings changes and then removing this line.